frontend/src/components/Search.tsx

```diff
@@ -76,6 +67,17 @@ const Search = (props: any) => {
   const [letterCodes, setLetterCodes] = useState<any[]>([]);
   const [queryString, setQueryString] = useState<string>('');
   const [numberRange, setNumberRange] = useState<number[]>([200, 400]);
+  const [names, setNames] = useState<string[]>([]);
+
+  useEffect(() => {
+    axios.get('/courses/letterCodes')
+      .then((response: AxiosResponse) => {
+        setNames(response.data);
+      })
+      .catch(() => {
+        console.log('error fetching course letter codes');
```

Comment:
If this is supposed to stay in prod, maybe make it console.warn
Otherwise, maybe just remove

Reply:
sure yea i'll make it console.warn